src/chatlog/chatline.h

```diff
@@ -70,13 +70,13 @@ class ChatLine
     void replaceContent(int col, ChatLineContent* lineContent);
     void layout(qreal width, QPointF scenePos);
     void moveBy(qreal deltaY);
-    void removeFromScene();
-    void addToScene(QGraphicsScene* scene);
-    void setVisible(bool visible);
-    void selectionCleared();
-    void selectionFocusChanged(bool focusIn);
+    void removeFromScene() const;
+    void addToScene(QGraphicsScene* scene) const;
+    void setVisible(bool visible) const;
```

Comment:
The above three methods are no const methods, because they alter the object or it's contents.